src/twisted/newsfragments/9453.bugfix

```diff
@@ -0,0 +1 @@
+twisted.internet._sslverify.KeyPair.generate: No longer generate 1024-bit RSA keys by default. Anyone who generated a key with this method using the default value should move to replace it immediately.
```

Comment:
The public name of the API is twisted.internet.ssl.KeyPair.

Reply:
Also the style guideline for news fragments suggests this should be more like:

    twisted.internet.ssl.KeyPair.generate now defaults to generating a 2048 bit key.

I doubt that this is an effective place to provide "immediate" security action guidance.